packages/gatsby-plugin-goober/gatsby-ssr.js

```diff
@@ -1,9 +1,22 @@
 const React = require('react');
 const goober = require('goober');
-const autoprefixer = require('goober-autoprefixer');
+
+const moduleExists = (path) => {
+    try {
+        require.resolve(path);
+        return true;
+    } catch (_) {
+        return false;
+    }
+};
+
+// Support new and old goober versions
+const autoprefixer = moduleExists('goober/prefixer')
+    ? require('goober/prefixer').prefix
+    : require('goober-autoprefixer').default;
 
 // Set the pragma for `goober` to know which to use
-goober.setup(React.createElement, autoprefixer.prefix);
+goober.setup(React.createElement, autoprefixer);
```

Comment:
Since we don't pass any theme or filter here function, does that mean this functionality can't be used?

Reply:
Hey @devrnt sorry, been away for a while. That's a good point. 🤔  No idea. At first glance I think you are right, wondering if this is something that the community is using though? 

Recently I believe the ecosystem is moving towards using css vars instead of theme with context.

Reply:
No rush @cristianbote 😀. Yes indeed, but I still believe that a lot of people are looking for such a "global theme pragma" since they are migrating from another css-in-js.

Anyway it's out of the question here, I just came across this while I was looking at the gatsby starter